tests/test_single_topology.py

```diff
@@ -39,6 +39,8 @@ def test_phenol():
     are present when either a single root anchor is provided, or when a root anchor and a neighbor anchor is provided.
     """
     mol = Chem.AddHs(Chem.MolFromSmiles("c1ccccc1O"))
+    AllChem.EmbedMolecule(mol, randomSeed=2022)
```

Comment:
Good to remember that `EmbedMolecule` is nondeterministic! (just noticing that we have several instances in this in other tests that do not pass a seed)

Reply:
Based on a quick scan I didn't see any existing examples in the test where non-deterministic conformer generation would cause a problem, but good to keep in mind.

(This PR did introduce a test with a dependence on an aspect of the generated conformer that we have no reason to be conserved across RDKit updates, so it was important to freeze the conformer used by this PR's new tests -- https://github.com/proteneer/timemachine/pull/908#discussion_r1020432268 )